src/desktop/components/fair_layout/client/header.coffee

```diff
@@ -22,23 +22,27 @@ module.exports = class FairHeaderView extends Backbone.View
 
   openAuth: (options) ->
     if !sd.NEW_AUTH_MODAL
-      @modal = new AuthModalView _.extend({ width: '500px' }, options)
+      @modal = new AuthModalView options
```

Comment:
I know this is supporting the old auth modal, but I'm curious why we don't use the mediator here as well versus instantiating the view? Maybe we are using that `@modal` reference elsewhere?